dragonfly/implementations/postgres_interface.py

```diff
@@ -351,3 +382,16 @@ def _connect_id_table(self):
             self.it = sqlalchemy.Table('endpoint_id_map',self.provider.meta, autoload=True, schema=self.schema)
         except DriplineDatabaseError as dripline_error:
             logger.error('{}; when establishing connection to the "endpoint_id_map" table'.format(dripline_error.message))
+
+    def _get_endpoint_id(self, endpoint):
+        '''
+        Queries database to match endpoint to endpoint id
+        '''
```

Comment:
add a logger info explaining what is going to happen. there is a raise in the method which can stop it before tell we are using it